geotrellis-python/README.md

```diff
@@ -0,0 +1,43 @@
+GeoTrellis uses [Apache Avro](https://avro.apache.org/) serialization when storing its tiled map layers. While GeoTrellis is a Scala project Avro provides libraries for multiple languages, including python. This project is to implement a python library that will read and write persisted GeoTrellis layers and present them as [NumPy](http://www.numpy.org/) datatypes that can be used by [rasterio](https://github.com/mapbox/rasterio). Once the basic translation is supported the next step would be to read GeoTrellis layer in a distributed Python Spark job and use rasterio to perform geospatial computations.
+
+Interoperability between GeoTrellis and rasterio will allow for complex, multi-stage geospatial workflows that do not suffer the usual performance penalties when working mixed Spark Scala/Python environment, expanding the utility of both libraries.
+
+#### Dependencies
+
+- [Apache Avro](https://avro.apache.org/)
+- [Avro Json Serializer](https://github.com/linkedin/python-avro-json-serializer)
+- [NumPy](http://www.numpy.org/)
+- [rasterio](https://github.com/mapbox/rasterio)
+- [pytz](https://github.com/newvem/pytz)
+- [pypng](https://github.com/drj11/pypng)
+- [Spec](https://github.com/bitprophet/spec) for testing
+
+#### Notes on compatibility
+
+The library will work with the following versions of dependencies (the command is `python setup.py install` under the main folder of the repo):
+- https://github.com/shiraeeshi/spec/tree/feature/options
+- https://github.com/shiraeeshi/python-avro-json-serializer/tree/feature/fullname
+- https://github.com/shiraeeshi/avro/tree/temp/with180version (The `setup.py` file resides in `lang/py` subdirectory)
```

Comment:
Good point, there is an issue in jira, which seems relevant (https://issues.apache.org/jira/browse/AVRO-1777), I've commented there.